_includes/scenarios/integrate/create-own-identityattribute.md

```diff
@@ -0,0 +1,111 @@
+{: .notice--info}
+
+If the Request is not made correctly, an [error message]({% link _docs_integrate/error-codes.md %}) is sent in response. Otherwise, the [Create an Attribute]({% link _docs_use-cases/use-case-consumption-create-an-attribute.md %}) use case occurs. In this case it is checked whether the input values for the properties of the specified IdentityAttribute Value meet the validation criteria documented on the [IdentityAttribute Values]({% link _docs_integrate/attribute-values.md %}#identity-attributes) page. Assuming a successful validation, the IdentityAttribute to be created is saved as a [LocalAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute). If it is a simple IdentityAttribute, a success response is sent directly. In the case of a complex IdentityAttribute, on the other hand, another LocalAttribute is created beforehand for each of its appropriate components.
+
+[![High level architecture diagram of enmeshed components and layers]({{ '/assets/images/integrate/Create own IdentityAttribute.svg' | relative_url }}){: .align-center}]({{ '/assets/images/integrate/Create own IdentityAttribute.svg' | relative_url }})
+
+<!--- Not magnifiable version: ![High level architecture diagram of enmeshed components and layers]({{ '/assets/images/integrate/Create own IdentityAttribute.svg' | relative_url }}){: .align-center} --->
+
+### Example 1: Create an own simple IdentityAttribute
+
+An example of a simple IdentityAttribute is an IdentityAttribute with IdentityAttribute Value of type [DisplayName]({% link _docs_integrate/attribute-values.md %}#displayname). If you want to create one without specifying optional parameters, the payload of the Request must have the following form:
+
+```json
+{
+  "content": {
+    "@type": "IdentityAttribute",
+    "owner": "<your Connector's Address>",
+    "value": {
+      "@type": "DisplayName",
+      "value": "<your Connector's display name>"
+    }
+  }
+}
+```
+
+Assuming that the input value for the Connector's display name meets the validation criterion, which means that the specified name is not more than 100 characters long, the IdentityAttribute is saved as a LocalAttribute and a success response is sent.
+
+### Example 2: Create an own complex IdentityAttribute
+
+An example of a complex IdentityAttribute is an IdentityAttribute with IdentityAttribute Value of type [BirthDate]({% link _docs_integrate/attribute-values.md %}#birthdate). If you want to create one without specifying optional parameters, the payload of the Request must have the following form:
```

Comment:
An example of a complex IdentityAttribute is one of the type [`BirthDate`]({% link _docs_integrate/attribute-values.md %}#birthdate).